primary/src/routes/manage/scoutingaudit.ts

```diff
@@ -7,6 +7,9 @@ import { upload as uploadHelper } from '@firstteam102/scoutradioz-helpers';
 import type { ImageLinks } from '@firstteam102/scoutradioz-helpers/types/uploadhelper';
 import e from '@firstteam102/http-errors';
 import type { MatchScouting, MatchTeamKey, Upload, Match, AnyDict, MatchFormData } from '@firstteam102/scoutradioz-types';
+import type Mathjs from 'mathjs';
+import { NumberLiteralType } from 'typescript';
```

Comment:
What's this for?

Reply:
Ah, good catch, that was a leftover import from getting mathjs.lusolve() to work. Removed in the newest commit.